Hunting Queries/SecurityEvent/ADAccountLockouts.yaml

```diff
@@ -0,0 +1,20 @@
+id: e7642e6e-cf27-46ec-a4b9-e4475228fead
+name: AD Account Lockout
+description: |
+  'Detects Active Directory account lockouts'
+requiredDataConnectors:
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - InitialAccess
+  - CredentialAccess
+relevantTechniques:
+  - T1531
+query: |
+  let timeframe = 7d;
+  SecurityEvent
+  | where TimeGenerated >= ago(timeframe)
+  | where EventID == 4740
+  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), count() by Account
+  | extend Timestamp = StartTimeUtc
```

Comment:
The entity extension here is used for entity mapping in the Azure Sentinel UX.  The value should be timestamp = StartTime, notice the lower case "t".  Also, if you could add in AccountCustomEntity = Account and HostCustomEntity = Computer that helps for mapping also.

Reply:
i will update the query as requested. again, please, just give me some time ;)